src/components/AgreementSections/ControllingLicense.js

```diff
@@ -77,14 +85,11 @@ export default class ControllingLicense extends React.Component {
           </KeyValue>
         }
         { currentAmendments.length ?
-          <KeyValue label={<FormattedMessage id="ui-agreements.license.currentAmendments" />}>
-            <LicenseAmendmentList
-              amendments={currentAmendments}
-              id="controlling-license-current-amendments"
-              license={licenseRecord}
-              renderStatuses
-            />
-          </KeyValue>
+          this.renderAmendments(linkedLicense, statuses.CURRENT)
```

Comment:
Looking at this...doesn't it seem as though we can just skip the generation of `currentAmendments` in this function and just always call `this.renderAmendments(linkedLicense, statuses.CURRENT)`? I don't really see where we're using `currentAmendments`

Reply:
We're using it right now to check if there are any or not, to check whether to render ```{ currentAmendments.length ?
          this.renderAmendments(linkedLicense, statuses.CURRENT)
          : null
        }```

Reply:
Right but check the definition of `renderAmendments` - if we call it unconditionally it'll do the same thing, right?

Reply:
Oh, damn, you're right. Same goes for `unsetAmendments` then I guess

Reply:
Yeah I think it's fine to leave it as-is for the unset amendments. It's more obvious what we're doing there then.